hs_core/views/__init__.py

```diff
@@ -719,30 +758,40 @@ def create_resource(request, *args, **kwargs):
     res_title = request.POST['title']
 
     resource_files = request.FILES.getlist('files')
-
+    fed_res_file_names=[]
     irods_fnames = request.POST.get('irods_file_names')
+    federated = True if request.POST.get("irods_federated").lower()=='true' else False
+    do_copy = request.POST.get("copy-or-move")
+    fed_copy = True
+    if do_copy:
```

Comment:
@pkdash this ```if``` is needed since "copy-or-move" may not exist in the request if no file is selected from a federated zone for creating resource request, in which case, do_copy could be None, so if ```if``` is needed here to handle this case.